web/src/main/java/no/nav/foreldrepenger/fordel/web/app/rest/journalføring/JournalføringRestTjeneste.java

```diff
@@ -295,9 +296,19 @@ private JournalpostDetaljerDto.BrukerDto mapBruker(String aktørId) {
     }
 
     private OppgaveDto lagOppgaveDto(Oppgave oppgave) {
-        return new OppgaveDto(Long.valueOf(oppgave.id()), oppgave.id(), oppgave.aktoerId(), hentPersonIdent(oppgave).orElse(null),
-            mapYtelseType(oppgave), oppgave.fristFerdigstillelse(), OppgavePrioritet.NORM, oppgave.beskrivelse(),
-            tekstFraBeskrivelse(oppgave.beskrivelse()), oppgave.aktivDato(), oppgave.tildeltEnhetsnr(), oppgave.tilordnetRessurs(), mapKilde(oppgave));
+        return new OppgaveDto(
+            Long.valueOf(oppgave.oppgaveId()),
+            oppgave.journalpostId(),
+            oppgave.aktørId(),
+            hentPersonIdent(oppgave).orElse(null),
+            mapYtelseType(oppgave),
+            oppgave.fristFerdigstillelse(),
+            OppgavePrioritet.NORM,
+            oppgave.beskrivelse(),
```

Comment:
Formattering av evt Gosys-oppgaver som vi fortsatt får inn.

Reply:
Brukes ikke i frontend lenger. Ble fjernet for lenge siden.